src/kafka-extensions/registry.ts

```diff
@@ -0,0 +1,51 @@
+import { Kafka } from "kafkajs";
+import { ConnectionOptions } from "../client/client";
+import { DefaultClusterWizardProvider } from "../wizards/clusters";
+import { ClusterProvider, KafkaClientFactory } from "./api";
+
+const defaultClusterProviderId = 'default';

```

Comment:
upper case

Reply:
I cannot do that because it breaks tslint rules.